packages/metals-vscode/src/releaseNotesProvider.ts

```diff
@@ -66,6 +66,8 @@ async function showReleaseNotesImpl(
   // below are helper functions
 
   async function showPanel(version: string, releaseNotesUrl: string) {
+    const releaseNotes = await getReleaseNotesMarkdown(releaseNotesUrl);
```

Comment:
Should we add some sort of timeout on this?

Reply:
Added!